test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -876,7 +1001,14 @@ int run_test_with_multi_import_same_ctx(
                     }
                     if (iter != (maxIter - 1))
                     {
-                        clCl2VkExternalSemaphore->signal(cmd_queue1);
+                        if (use_fence)
+                        {
+                            fence->wait();
```

Comment:
Need clFinish here instead of fence wait to ensure OpenCL work is done before starting vulkan work submission.